### ISSUE TITLE
Possible to take a photo with camera and draw on to it?

### ISSUE THREAD

**Issue by brendanmullan** [acerbetti/ACEDrawingView#46]
Just quick question, would it be possible to take a photo with the device camera and draw onto it? Might be a nice feature to add :-)

**Comment by TGNA** [comment]
+1

**Comment by ViccAlexander** [comment]
Yes it is. You can use something like [LLSimpleCamera](https://github.com/omergul123/LLSimpleCamera) to setup the capturing feature. Once a photo has been taken, you can send the photo to a new `UIViewController` (where you'll edit the image / draw on it). You can set the default view's class to `ACEDrawingView` in the editing controller, and pass the captured photo as it's background image. From there everything is straightforward. :sunglasses:

**Comment by ntnmrndn** [comment]
Actually it's not working properly...  See <img width="880" alt="capture d ecran 2015-10-19 a 17 47 13" src="https://cloud.githubusercontent.com/assets/1104823/10582712/76173178-7689-11e5-8808-65e0c5171335.png">

Related : https://github.com/acerbetti/ACEDrawingView/issues/27

**Comment by ntnmrndn** [comment]
Actually it seems to work if you add the picture in "viewDidLayoutSubview" AND if the picture is exactly the same size as the view.

**Comment by hollyfeld** [comment]
Set the ACEDrawingModeScale on ACEDrawingView to ACEDrawingModeScale.

**Comment by deviserRahul** [comment]
I have made some changes for drawing on image. Please have a look [here](https://github.com/deviserRahul/ACEDrawingView)

**Comment by acerbetti** [comment]
Do you mind to do a PR ?

**Comment by deviserRahul** [comment]
What exactly will be requiring to do in PR?

On Sat, Oct 8, 2016 at 10:17 AM, Stefano Acerbetti <notifications@github.com

**Comment by deviserRahul** [comment]
Please review the request, suggest improvements.
